ssz/simple-serialize.md

```diff
@@ -227,7 +241,7 @@ We first define helper functions:
         - If `1` chunk: the root is the chunk itself.
         - If `> 1` chunks: merkleize as binary tree.
 * `mix_in_length`: Given a Merkle root `root` and a length `length` (`"uint256"` little-endian serialization) return `hash(root + length)`.
-* `mix_in_type`: Given a Merkle root `root` and a type_index `type_index` (`"uint256"` little-endian serialization) return `hash(root + type_index)`.
+* `mix_in_selector`: Given a Merkle root `root` and a type selector `selector` (`"uint256"` little-endian serialization) return `hash(root + selector)`.
```

Comment:
```suggestion
* `mix_in_selector`: Given a Merkle root `root` and a type selector `selector` (`"uint8"` little-endian serialization) return `hash(root + selector)`.
```

do you mean to refer to the root or the selector in the parenthetical? a bit confused what it is referring to

Reply:
`hash_tree_root(uint8(selector))` is the same as `uint256(selector)`. We convert the selector to 32 bytes before hashing it together with the contents `root`, so it's consistent with the `(32, 32) -> 64` byte merkleization everywhere.